src/include/ipc/stream.h

```diff
@@ -92,8 +92,9 @@ struct sof_ipc_stream_params {
 
 	/* for notifying host period has completed - 0 means no period IRQ */
 	uint32_t host_period_bytes;
+	uint16_t period_irq;
 
-	uint32_t reserved[2];
+	uint16_t reserved[3];
```

Comment:
you also have a comment that needs to be fixed line 93 and in general it helps when you add more details on a boolean. in other places we use no_irq, here it's period_irq so 50% of people will get it wrong :-)

Reply:
@mrajwa is this now resolved ? We can merge if resolved.

Reply:
@lgirdwood yes I think we can merge both sides simultaneously.